R/binning.R

```diff
@@ -9,7 +9,8 @@
 #' @param x `numeric` with the values that should be aggregated/binned.
 #'
 #' @param y `numeric` with same length than `x` with values to be used for
-#'     the binning.
+#'     the binning. Ideally, `y` should be increasingly sorted, but it's not 
+#'     mandatory.
```

Comment:
I am somehow against silently reordering `y`. Please see https://github.com/rformassspectrometry/MsCoreUtils/issues/108#issuecomment-1382383808

Reply:
I'm with @sgibb here and advise against silently re-ordering.

Reply:
Yes, agree. Please no silent re-ordering (see also https://github.com/rformassspectrometry/MsCoreUtils/issues/108#issuecomment-1400453934)